codelabs/Module2_APITesting.md

```diff
@@ -132,61 +147,80 @@ From the Intersitial page:
    "HTTP Code 401: Authorization failed"
    ```
    <img src="assets/apif-mod2/401-error.png" alt="API Fortress: HTTP 401 Request Error" />
+
+
 #### Note:
 
 Negative
 : To see the raw response body in the HTTP Client select **Body** and then either *Raw* or *Parsed*
 
 ### Adding a Test Component
+Lets add a GET component that can make a successful API call against the Sauce Labs REST API.
 
-In the previous step we encountered a common authentication error—[HTTP: 401](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401). In order to fix this error we must add our [Sauce Labs Account Credentails](https://app.saucelabs.com/user-settings) as a [Basic Authentication Header](https://docs.saucelabs.com/api-testing/how-to/authorization-simple-oauth-etc/index.html#overview).
+In the previous step we encountered a common authentication error **[HTTP: 401](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401**) since you didn't send the username and access key that Sauce Labs requires to communicate with their API. In order to fix this error we must add our [Sauce Labs Account Credentails](https://app.saucelabs.com/user-settings) as a [Basic Authentication Header](https://docs.saucelabs.com/api-testing/how-to/authorization-simple-oauth-etc/index.html#overview).
```

Comment:
```suggestion
In the previous step we encountered a common authentication error, **[HTTP: 401](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401**), since you didn't send the username and access key that Sauce Labs requires to communicate with their API. In order to fix this error we must add our [Sauce Labs Account Credentails](https://app.saucelabs.com/user-settings) as a [Basic Authentication Header](https://docs.saucelabs.com/api-testing/how-to/authorization-simple-oauth-etc/index.html#overview).
```